run_tests.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import feedparser
+import sys
+import os
+
+feed = feedparser.parse('./feed.rss')
+
+absolute_address = "https://contravariance.rocks/"
+
+def validate_path(path):
+    replaced = path.replace(absolute_address, "")
+    # If file doesn't exist or is empty, return false
+    return os.path.getsize(replaced) != 0
+
+def fail(message):
+    print message
+    sys.exit(1)
+
+for entry in feed.entries:
+    print "Validating", entry.title
+    if not validate_path(entry["image"]["href"]):
+        fail("Could not find image %s" % (entry["image"]))
+    for enclosure in entry.enclosures:
+        if int(enclosure["length"] == 0):
+            fail("Invalid length for item '%s'" % (entry.title,))
```

Comment:
What does this this trailing comma syntax do?

Reply:
telling python that it is a tuple, and not an expression (i.e. `(3, 2, 1)` vs `3 * (2 + 1)`) I'm not sure if it is required, but I picked it up more than a decade ago and it is difficult to let go